xcp_d/interfaces/bids.py

```diff
@@ -165,28 +166,40 @@ def _run_interface(self, runtime):
                 f"{hstr}.sphere.reg2",
             )
 
-            # TODO: Collect from templateflow once it's uploaded.
-            # MCRIBS: tpl-fsaverage_hemi-?_den-41k_desc-reg_sphere.surf.gii
-            self._results["source_sphere"] = os.path.join(
-                self.inputs.segmentation_dir,
-                "templates_fsLR",
-                f"tpl-fsaverage_hemi-{hemisphere}_den-41k_desc-reg_sphere.surf.gii",
+            # Load the fsaverage-41k sphere.
+            self._results["source_sphere"] = str(
+                get_template(
+                    template="fsaverage",
+                    space=None,
+                    hemi=hemisphere,
+                    density="41k",
+                    desc="reg",
+                    suffix="sphere",
+                )
             )
 
-            # TODO: Collect from templateflow once it's uploaded.
-            # MCRIBS: tpl-dHCP_space-fsaverage_hemi-?_den-41k_desc-reg_sphere.surf.gii
-            self._results["sphere_to_sphere"] = os.path.join(
-                self.inputs.segmentation_dir,
-                "templates_fsLR",
-                f"tpl-dHCP_space-fsaverage_hemi-{hemisphere}_den-41k_desc-reg_sphere.surf.gii",
+            # Load the fsaverage-to-dhcpSym(?) warp sphere.
+            self._results["sphere_to_sphere"] = str(
+                get_template(
+                    template="dhcpSym",
+                    space="fsaverage",
+                    hemi=hemisphere,
+                    density="41k",
+                    desc="reg",
+                    suffix="sphere",
+                )
             )
```

Comment:
We need cohorts here. Also, I believe we decided not to add the warp spheres when adding the templates to TemplateFlow, but it would be good to get them on there soon.